src/index.js

```diff
@@ -1,104 +1,243 @@
-        let facChoice = await read("");
-        switch (facChoice) {
-          case "0":
-            faction = factions[0].name;
-            facId = factions[0].id;
-            break;
-            case "1":
-              faction = factions[1].name;
-              facId = factions[1].id;
-              break;
-            case "2":
-              faction = factions[2].name;
-              facId = factions[2].id;
-            break;
-          default:
-            console.error("Chose a valid option");
-            break;
+        facChoice = await read("");
+        facChoice--;
+        if (facChoice >= 0 && facChoice < 3) {
+          return [factions[facChoice].name, factions[facChoice].id];
+        } else {
+          clear();
+          console.error("Chose a valid option.");
         }
-        console.log(`Welcome ${faction} dev!`);
+        facChoice++;
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
 
-        const name = await read("What is your name?\n"); //example for read user input
-        console.log(`Nice to meet you ${name}!`);
-        let user = {
-          "atk": 10,
-          "def": 10,
-          "agi": 10,
-          "hp": 10,
-          "gold": 100,
-          "name": name,
-          "factions": [
-            {
-              "factions_id": {
-                "id": facId,
-                "name": faction
-              }
-            }
-          ],
-          "equipments": []
-        };
-        try{
-          const print = await postApi(user, "characters?fields=*,equipments.equipments_id.*,factions.factions_id.*")
-          console.log(print);
-        } catch (error){
-          console.log(error);
-        }
+  const quests = async () => {
+    clear();
+    console.log("select a quest:\n");
+    let questList = await getApi("tasks");
+    let bugList = await getApi("bugs");
+    questList.forEach((questn, index) => {
+      let bugHp = [1];
+      questn.bugs.forEach((monstern) => {
+        let bugn = bugList.filter((bug) => bug.id == monstern);
+        bugHp.push(bugn[0].hp);
+      });
+      bugHp.shift();
+      console.log(
+        `${index + 1}: ${questn.name}\nDescription: ${
+          questn.description
+        }\nBug hp: ${bugHp}\nReward: ${questn.reward}\n`
+      );
+    });
+  };
+
+  const store = async (costumer) => {
+    clear();
+    console.log(
+      `You want it? It's your's my friend! As long as you have enough gold.\nGold: ${costumer.gold}\n`
+    );
+    let equipList = await getApi("equipments");
+    let buy = 1;
+    while (buy != 0) {
+      equipList.forEach((equipn, index) => {
+        console.log(
+          `${index + 1} - ${equipn.name}\nPrice: ${equipn.value} gold\nStat: +${
+            equipn.affected_amount
+          } ${equipn.affected_attribute}\n`
+        );
+      });
+      buy = await read("0 - Exit");
+      if (buy == 0) {
+        clear();
+        console.log("Take care out there my friend!");
+        return 0;
```

Comment:
not sure if you need this function to return a value, but an alternative to `break` the loop, would be the `break` command